src/Installer/redist-installer/targets/GenerateMSIs.targets

```diff
@@ -319,9 +319,11 @@
     </ItemGroup>
     <PropertyGroup>
       <LatestTemplateMsiInstallerFile>@(LatestTemplateInstallerComponent->'%(MSIInstallerFile)')</LatestTemplateMsiInstallerFile>
+      <UpgradePoliciesSrcPath>$(PkgMicrosoft_DotNet_Build_Tasks_Installers)\build\wix\bundle\upgradePolicies.wxs</UpgradePoliciesSrcPath>
```

Comment:
I assume this file comes from the arcade flow and as long as the build works, we know this is working right?

Reply:
Correct. Arcade is the source of truth - this avoid needing to make fixes everywhere. We only need to update Arcade and wait for it to flow and everyone gets the change